packages/gatsby-plugin-subfont/src/gatsby-node.js

```diff
@@ -1,22 +1,23 @@
 const path = require(`path`)
-const { execSync } = require(`child_process`)
-const shellescape = require(`shell-escape`)
+const subfont = require(`subfont`)
 
-exports.onPostBuild = ({ store }) => {
+exports.onPostBuild = async ({ store }, options) => {
   const root = path.join(store.getState().program.directory, `public`)
-  // TODO make this configurable
+
   const urlPaths = [`/`]
-  const baseArgs = [
-    `node_modules/.bin/subfont`,
-    `-i`,
-    `--no-recursive`,
-    `--inline-css`,
-    `--root`,
-    `file://${root}`,
-  ]
-  const args = baseArgs.concat(
-    urlPaths.map(currentPath => path.join(root, currentPath, `index.html`))
+
+  await subfont(
+    {
+      root: `file://${root}`,
+      inPlace: true,
+      inlineCss: true,
+      silent: true,
+      inputFiles: urlPaths.map(currentPath =>
+        path.join(root, currentPath, `index.html`)
+      ),
```

Comment:
If the `urlPaths` is only ever going to have one element we should simplify this until it doesn't...
(Also, `path.join` will infix the correct platform dependent slash so you don't need to)

Is there any chance you could add a simple test for this to confirm (through the CI) that this also works on Windows paths?

```suggestion
      inputFiles: [path.join(root, `index.html`)],
```

Reply:
Fixed. thanks